src/models/table-options.model.ts

```diff
@@ -53,9 +53,6 @@ export class TableOptions {
   // Page offset
   offset: number = 0;
 
-  // Loading indicator
-  loadingIndicator: boolean = false;
```

Comment:
Sorry that was a mistake. Meant to delete loadingMessage, which is not used.

Will fix that and let you know when all set.